src/phalanx/models/environments.py

```diff
@@ -205,6 +205,15 @@ class EnvironmentBaseConfig(CamelCaseModel):
         ),
     )
 
+    namespace_labels: dict[str, Any] | None = Field(
```

Comment:
Any shouldn't be used in types. Right now this is `dict[str, dict[str, str]]`.

Reply:
Linter didn't like that one. Likes `dict[str, str]` instead, but not sure that's right either.

Reply:
Oh, right, sorry, I had one too many layers there.